benchmark/run_benchmark.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+# Copyright (c) 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# In this benchmark case, precision can be: bf16, bf16_fp16, fp16, bf16_int8, int8 
+precision="fp16"
+# The model name can be choosen are: llama-2(-7b,-13b,-70b), llama(-7b,-13b,-30b,-65b), chatglm2-6b, chatglm-6b, baichuan2(-7b,13b)
+# which you can refer to ../examples/model_config as well
+model_name="chatglm2-6b"
```

Comment:
I exposed them as env variables. Does that make seance?

Reply:
It's ok. But it's not very user-friendly for normal users, people are familiar with parameters. Moreover, model_name and dtype should be required parameters. In addition, it is necessary to print model_name and data type, otherwise, users may not be aware of which case they are running.

Reply:
Switched to getopts.